static/js/components/search/NestedAggregatingMenuFilter.js

```diff
@@ -0,0 +1,292 @@
+import PatchedMenuFilter from './PatchedMenuFilter';
```

Comment:
no, we don't have any tests for searchkit components yet. we probably should, but i don't think we want to get into that as part of this bug fix. @pdpinch you can chime in if you like